Purchases/SubscriberAttributes/SubscriberAttributesManager.swift

```diff
@@ -248,13 +230,11 @@ private extension SubscriberAttributesManager {
         return maybeAttribute?.value
     }
 
-    func setAttributionID(networkID: String?, networkKey: String, appUserID: String) {
+    func setAttributionID(_ attributionID: String?,
+                          forNetworkID networkID: ReservedSubscriberAttribute,
+                          appUserID: String) {
         collectDeviceIdentifiers(forAppUserID: appUserID)
-        setAttribute(key: networkKey, value: networkID, appUserID: appUserID)
-    }
-
-    func logAttributionMethodCalled(functionName: String) {
-        Logger.debug(Strings.attribution.method_called(methodName: functionName))
+        setAttribute(key: networkID.key, value: attributionID, appUserID: appUserID)
```

Comment:
I think this should be `setReservedAttribute`, otherwise we would miss the log

Reply:
yes!! This is why I shouldn't do refactors in the early morning

Reply:
updated!